src/components/Kv/KvLightbox.vue

```diff
@@ -16,43 +16,45 @@
 				class="kv-lightbox__focus-lock"
 				:return-focus="true"
 			>
-				<div
-					class="kv-lightbox__container"
-					@click.stop
-					tabindex="-1"
-					ref="kvLightbox"
-					role="dialog"
-					data-test="kv-lightbox"
-					:aria-labelledby="title ? 'lightbox-title' : null"
-					aria-modal="true"
-				>
-					<div class="kv-lightbox__header">
-						<h2 v-if="title"
-							class="kv-lightbox__title"
-							id="lightbox-title"
-						>
-							{{ title }}
-						</h2>
-						<button
-							v-if="!preventClose"
-							class="kv-lightbox__close-btn"
-							@click.stop.prevent="closeLightbox"
-							aria-label="Close"
-						>
-							<kv-icon
-								class="kv-lightbox__close-btn-icon"
-								name="small-x"
-								:from-sprite="true"
-							/>
-						</button>
-					</div>
-					<div class="kv-lightbox__body"
-						ref="kvLightboxBody"
+				<div class="kv-lightbox__container">
```

Comment:
This is the main change. Adds a wrapper div to hook into because <focus-lock> was adding a div outside of normal control.